# HG changeset patch
# User Rik <rik@octave.org>
# Date 1519949750 28800
#      Thu Mar 01 16:15:50 2018 -0800
# Node ID 89b62d22d7231f6265b1f22541c460a705260b4d
# Parent  f45e2c6dc3e051aa410a16bf628132dd52885b10
configure.ac: Delete check for Xft library.

* configure.ac: Delete check for Xft library which is not used directly by
Octave.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2066,27 +2066,16 @@ if test -n "$warn_freetype"; then
 else
   FT2_CPPFLAGS="$FT2_CFLAGS"
   dnl Alias CPPFLAGS to CFLAGS.  This is closer to the true meaning
   dnl of `pkg-config --cflags` output.
   FT2_CPPFLAGS="$FT2_CFLAGS"
   AC_SUBST(FT2_CPPFLAGS)
 fi
 
-## Check for Xft library (when using X11)
-
-dnl FIXME: We don't use this library directly, so there's no need to
-dnl check for the header file.  Do we even need to check for the library?
-
-OCTAVE_CHECK_LIB(Xft, Xft,
-  [Xft library not found.  OpenGL graphics will not be fully functional.],
-  [], [XftFontOpen],
-  [], [don't use Xft library, OpenGL graphics will not be fully functional],
-  [], [xft])
-
 ## Check for fontconfig library
 
 OCTAVE_CHECK_LIB(fontconfig, fontconfig,
   [Fontconfig library not found.  OpenGL graphics will not be fully functional.],
   [fontconfig.h fontconfig/fontconfig.h], [FcInit],
   [], [don't use fontconfig library, OpenGL graphics will not be fully functional])
 
 ## Check for gl2ps which is required for printing with OpenGL graphics.
